src/abi/abitrumBridgeAbi.ts

```diff
@@ -0,0 +1,9 @@
+export const ARBITRUM_BRIDGE_ABI = [
+  {
+    inputs: [{ internalType: 'address', name: 'l1ERC20', type: 'address' }],
+    name: 'calculateL2TokenAddress',
```

Comment:
Different than Omnibridge contract, this one always return an address. We might add tokens which have not been bridged yet.